### PR TITLE
Changing CLI group leaders and adding mailing list

### PR DESCRIPTION
**Pull request by spacekookie**
Please don't merge yet, this is a temp PR that we will discuss further as a group next week.

### CODE REVIEW

**Comment by JohnTitor** [comment]
Just note that you should remain in members array.

**Comment by spacekookie** [comment]
Woops, yea that makes sense :sweat_smile: I'll rebase this after our meeting

File: teams/wg-cli.toml [hunk 1]
```diff
@@ -2,21 +2,23 @@ name = "wg-cli"
...
 page = "cli"
-name = "Command-line interfaces (CLI) working group"
-description = "Focusing on the end-to-end experience of writing CLI apps, both large and small, in Rust."
+name = "Terminal applications  working group"
+description = "Focusing on the end-to-end experience of writing terminal apps, both large and small, in Rust."
 discord-invite = "https://discord.gg/BXCBYX6"
 discord-name = "#wg-cli"
+
+[[lists]]
+address = "embedded-wg@rust-lang.org"
```
Comment (matthiasbeyer) [inline teams/wg-cli.toml]: I guess this is a copy-paste artifact here, is it? :smile:

Comment (spacekookie) [reply teams/wg-cli.toml]: Yes :see_no_evil:

**Comment by spacekookie** [comment]
I used sort(1) to sort the contributors instead of doing it by hand, meaning that capital letters before lower case, and also that it becomes more reproducible to sort lists this way.

We might add more people to the group, or "extra people" to read the ML, but this should be good for now

**Comment by spacekookie** [comment]
:crossed_fingers: Well that should be it now then

File: people/matthiasbeyer.toml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 name = "Matthias Beyer"
+email = "mail@matthiasbeyer.de"
```
Comment (matthiasbeyer) [inline people/matthiasbeyer.toml]: This is not my E-Mail address!

Comment (matthiasbeyer) [reply people/matthiasbeyer.toml]: I just send out a fixup patch to @spacekookie for this.

Comment (spacekookie) [reply people/matthiasbeyer.toml]: You made me realise I need a better workflow for applying patches to a project directly in emacs with notmuch :see_no_evil:

**Pull request merged**